pkg/client/cache/ssdb/ssdb.go

```diff
@@ -24,65 +26,66 @@ func NewSsdbCache() cache.Cache {
 }
 
 // Get gets a key's value from memcache.
-func (rc *Cache) Get(key string) interface{} {
+func (rc *Cache) Get(key string) (interface{}, error) {
+	return rc.GetWithCtx(context.Background(), key)
+}
+
+// GetWithCtx gets a key's value from memcache.
+func (rc *Cache) GetWithCtx(ctx context.Context, key string) (interface{}, error) {
+	rc.conn.Do()
 	if rc.conn == nil {
 		if err := rc.connectInit(); err != nil {
-			return nil
+			return nil, nil
 		}
 	}
-	value, err := rc.conn.Get(key)
-	if err == nil {
-		return value
-	}
-	return nil
+	return rc.conn.Get(key)
 }
 
 // GetMulti gets one or keys values from memcache.
-func (rc *Cache) GetMulti(keys []string) []interface{} {
+func (rc *Cache) GetMulti(keys []string) ([]interface{}, error) {
+	return rc.GetMultiWithCtx(context.Background(), keys)
+}
+
+// GetMultiWithCtx gets one or keys values from memcache.
+func (rc *Cache) GetMultiWithCtx(ctx context.Context, keys []string) ([]interface{}, error) {
 	size := len(keys)
 	var values []interface{}
 	if rc.conn == nil {
 		if err := rc.connectInit(); err != nil {
 			for i := 0; i < size; i++ {
-				values = append(values, err)
+				values = append(values, nil)
 			}
-			return values
+			return values, err
 		}
 	}
 	res, err := rc.conn.Do("multi_get", keys)
 	resSize := len(res)
 	if err == nil {
-		for i := 1; i < resSize; i += 2 {
-			values = append(values, res[i+1])
+		for i := 0; i < size; i++ {
+			values = append(values, nil)
 		}
-		return values
+		return values, err
 	}
-	for i := 0; i < size; i++ {
-		values = append(values, err)
+	for i := 1; i < resSize; i += 2 {
+		values = append(values, res[i+1])
 	}
-	return values
-}
-
-// DelMulti deletes one or more keys from memcache
-func (rc *Cache) DelMulti(keys []string) error {
```

Comment:
Why do we remove this method?

Reply:
Sorry, I deleted,  cache interface has not this method

Reply:
Keeping this method would be better because this is public method. And it could be used by current users. They may use type conversion to use this method like `rc.(*Cache).DelMulti()...`

And I think we need to support this method for all implementations in the future.

Reply:
@mlboy